src/cliffordsynthesis/encoding/SATEncoder.cpp

```diff
@@ -130,7 +141,6 @@ Results SATEncoder::run() {
   const auto start = std::chrono::high_resolution_clock::now();
 
   createFormulation();
-  produceInstance();
```

Comment:
has this been removed on purpose?

Reply:
Yes. `produceInstance` is invoked in the call to `solve` in the LogicBlocks library. Even worse, there was no check whether the instance was already created, basically doubling the size of the encoding without any benefit.

Reply:
Oh wow. What a waste.